File: arrays/basics/py/reverse.py

def reverse(arr):
    rev=[];
    i=0;
    j=len(arr);
    for i in range(len(arr)):
        rev.append(arr[len(arr) - 1 - i])
    return rev;

ary=[10,20,30,40,50];
print(f"Reverse : {reverse(ary)}")
    

File: arrays/basics/reverse.ts
const reverse = (arr: number[]): number[] => {
  let rev: number[] = [];
  let j = 0;
  for (let i = arr.length - 1; i >= 0; i--) {
    rev[j] = arr[i];
    j++;
  }
  return rev;
};

const arrReverse = [10, 20, 30, 40, 50, 50];
console.log(`Reveresed : ${reverse(arrReverse)} \n Original : ${arrReverse}`);
